src/docs/implementations/wavefront.adoc

```diff
@@ -35,25 +42,28 @@ WavefrontConfig config = new WavefrontConfig() {
 };
 MeterRegistry registry = new WavefrontMeterRegistry(config, Clock.SYSTEM);
 ----
-<1> "longboard" is the name of the co-tenant instance that most organizations start on. Once you reach a sufficient scale, Wavefront may move you
+<1> `longboard` is the name of the co-tenant instance on which most organizations start. Once you reach a sufficient scale, Wavefront may move you
```

Comment:
*"longboard"* instead?

Reply:
I think the code font (which the backticks create) is right here. I looked at Wavefront's docs, and they never use the term outside of a URL, and those are always marked as code.